src/frontend/src/app/shared/services/flow-structure.service.ts

```diff
@@ -69,7 +70,7 @@ export class FlowStructureService {
     this.positionsUpdate = true;
     const pipes = this.structure.pipes;
     const newForward =
-      '\n\t\t<Forward name="success" path="' + targetName + '" />';
+      '\n\t\t\t\t<Forward name="success" path="' + targetName + '" />';
```

Comment:
It feels like this has been changed a lot now, should we maybe look into detecting how much spacing it needs?
(this is just an idea)

Reply:
We should, but not a high priority feature.

Reply:
Allright!

Reply:
I'll add a new entry in the backlog